visallo/visallo
# Added: display user last login time on the user access page

Testing Instructions:
- Login as a new user, open user dialog, verify shows first login
- Logout and log back in, open user dialog verify last date is shown

CHANGELOG
Added: display user last login time on the user access page
